src/main/io/statusindicator.c

```diff
@@ -32,7 +32,9 @@ static uint32_t warningLedTimer = 0;
 typedef enum {
     WARNING_LED_OFF = 0,
     WARNING_LED_ON,
-    WARNING_LED_FLASH
```

Comment:
did you consiider https://github.com/cleanflight/cleanflight/pull/2201? 

This code is so ugly ...